src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Webservice/WebserviceKeyType.php

```diff
@@ -87,7 +87,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('key', GeneratableTextType::class, [
                 'label' => $this->trans('Key', 'Admin.Advparameters.Feature'),
-                'help' => $this->trans('Webservice account key.', 'Admin.Advparameters.Feature'),
+                'help' => sprintf(
+                    '%s<br>%s',
+                    $this->trans('Webservice account key.', 'Admin.Advparameters.Feature'),
+                    $this->trans(
+                        'Key should be at least %length% characters long.',
```

Comment:
This is not the right translation domain. It should be Admin.Advparameters.Help

See => https://zeroheight.com/80a6c5a61/v/latest/p/738e72-manage-translation-domains

We also miss the article "The" before "key".